Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,74 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+header{
+    background-color: white;
+    height: 100px;
+    }
+    
+    header img{
+        padding-top: 25px;
+    }
+    
+    .learnMore{
+        background: linear-gradient( rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5) ), url('/assets/images/flowershop.jpg');;
```

Comment:
This path should be the relative path from the index.css

../images/flowershop.jpg

Great use of linear gradient to apply the dark overlay!